nextcord/threads.py

```diff
@@ -830,3 +830,8 @@ def _from_data(self, data: ThreadMemberPayload):
     def thread(self) -> Thread:
         """:class:`Thread`: The thread this member belongs to."""
         return self.parent
+
+    @property
+    def member(self) -> Optional[Member]:
+        """Optional[:class:`Member`]: The member object of itself."""
```

Comment:
I think it sounds better like this
```suggestion
        """Optional[:class:`Member`]: The :class:`Member` object of this thread member."""
```

Reply:
this was never actually resolved @MaskDuck

Reply:
i thought i applied the suggestion. sorry.